src/sagemaker/model.py

```diff
@@ -114,6 +115,8 @@ def __init__(
                 network isolation in the endpoint, isolating the model
                 container. No inbound or outbound network calls can be made to
                 or from the model container.
+            output_kms_key (str): KMS key ARN, it's only used to encrypt the
+                repacked model archive file.
```

Comment:
"KMS key ARN used to encrypt the repacked model archive file if the model is repacked"

Reply:
k